lib/OpenAPI/Client.pm

```diff
@@ -37,8 +38,27 @@ has ua => sub { Mojo::UserAgent->new };
 sub call {
   my ($self, $op) = (shift, shift);
   my $code = $self->can($op);
-  return $self->$code(@_) if $code;
-  Carp::croak('[OpenAPI::Client] No such operationId');
+  Carp::croak('[OpenAPI::Client] No such operationId') if !$code;
+  $self->$code(@_);
+}
+
+sub call_p {
+  my ($self, $op) = (shift, shift);
+  my $code = $self->can($op);
+  my $promise = Mojo::Promise->new;
+  if (!$code) {
+    $promise->reject('[OpenAPI::Client] No such operationId');
```

Comment:
I would very much like if `call_p()` had the same behaviour as `call()`: Just croak.